Add walkthrough guide (#310)
HealthScreening/HealthScreeningBot: guides/walkthrough/page_2.md

> `set` : Sets additional information for the automated health screening.

Reviewer: ```suggestion
__**Other Useful Screening-Related Commands**__:
`/set`: Sets additional information for the automated health screening.
```

Paremeters (All optional by the way):

> - device: The name of the device for the health screening to "use". To view the full device list, do `/guide name:device_list`.

Reviewer: ```suggestion
- **device**: The name of the device for the health screening to "use". To view the full device list, do `/guide name:device_list`. *Note*: The device is used with `/generate_once` as well, so deleting your auto information will not completely reset your device. The default device used is `iPhone 11`.
```

> - hour : The hour to run the health screening at. Number should be in between 0 - 23.

Reviewer: ```suggestion
- **hour**: The hour to run the health screening at. Should be a number in the range of 0-23. *Note*: Changing the hour does not set the minute to zero. The default hour is 5.
```

> - minute : The minute to run the health screening at. Number  should be in between 0 - 59.

Reviewer: ```suggestion
- **minute**: The minute to run the health screening at. Should be a number between 0 - 59.
```

> *Note*: Together, the hour and time should represent the time you get your health screening at, such as 5:20, or 23:35 (or 11:35 PM)

Reviewer: ```suggestion
Together, the hour and minute should represent the time you get your health screening at, such as 5:20, or 23:35 (11:35 PM)
```

> - type : The type of screening to generate. Available options are __guest__, __student__, or __employee__, but defaults to __guest__.

Reviewer: ```suggestion
- **type**: The type of screening to generate. Available options are __guest__, __student__, or __employee__. The default is __guest__.
```

> - email_only : Whether the health screening should be emailed only or not. If this is True, the bot will **not** DM you the health screening on Discord. This should be either __True__ or __False__, but defaults to False.

Reviewer: ```suggestion
- **email_only**: Whether the health screening should be only emailed or emailed and screenshotted. If this is __True__, the bot will **not** automatically DM you the health screening on Discord. This should be either __True__ or __False__. The default is __False__. The `/generate_auto` command will produce a screenshot regardless of what this is set to.
```

> - paused : Whether to pause the auto health screenings. If this is True, you will **not** receive any automated screenings. This should be either __True__ or __False__, but defaults to False.

Reviewer: ```suggestion
- **paused**: Whether to pause the auto health screenings. If this is True, you will **not** receive any automated screenings, including emails. This should be either __True__ or __False__. The default is __False__.
```

> *Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If False, you will **not** receive any automated screenings on that day.

Reviewer: ```suggestion
*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If __False__, you will **not** receive any automated screenings on that day.
```

> *Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If False, you will **not** receive any automated screenings on that day.

Reviewer: ```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If False, you will **not** receive any automated screenings on that day.
```

> - sunday - monday - tuesday - wednesday - thursday - friday - saturday

Reviewer: ```suggestion
- **sunday**: Defaults to __False__.
- **monday**: Defaults to __True__.
- **tuesday**: Defaults to __True__.
- **wednesday**: Defaults to __True__.
- **thursday**: Defaults to __True__.
- **friday**: Defaults to __True__.
- **saturday**: Defaults to __False__.
```

> 
> 

Reviewer: ```suggestion

```

> `generate_once` : Generate a singular health screening without using your information set with `set_auto`.

Reviewer: ```suggestion
`/generate_once`: Generate a singular health screening without using your information set with `/set_auto`. If set, the device **will** be considered.
```

> Parameters:

Reviewer: ```suggestion
__Parameters__:
```

> - first_name : Your first name

Reviewer: ```suggestion
- **first_name**: Your first name.
```

> - last_name : Your last name

Reviewer: ```suggestion
- **last_name**: Your last name.
```

> - email : Your email 

Reviewer: ```suggestion
- **email**: Your email.
```

> - vaccinated : Whether or not you are vaccinated. This should be either __True__ or __False__

Reviewer: ```suggestion
- **vaccinated**: Whether or not you are vaccinated. This should be either __True__ or __False__.
```

> - ephemeral : Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__

Reviewer: ```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
```

__**Other Useful Screening-Related Commands**__:

> `/set`: Sets additional information for the automated health screening.

Reviewer: ```suggestion

`/set`: Sets additional information for the automated health screening.
```

> Paremeters (All optional by the way):

Reviewer: ```suggestion
__Parameters__ (All optional by the way):
```

- **hour**: The hour to run the health screening at. Should be a number in the range of 0-23. *Note*: Changing the hour does not set the minute to zero. The default hour is 5.
- **minute**: The minute to run the health screening at. Should be a number between 0 - 59.
Together, the hour and minute should represent the time you get your health screening at, such as 5:20, or 23:35 (11:35 PM)
- **type**: The type of screening to generate. Available options are __guest__, __student__, or __employee__. The default is __guest__.
- **email_only**: Whether the health screening should be only emailed or emailed and screenshotted. If this is __True__, the bot will **not** automatically DM you the health screening on Discord. This should be either __True__ or __False__. The default is __False__. The `/generate_auto` command will produce a screenshot regardless of what this is set to.
- **paused**: Whether to pause the auto health screenings. If this is True, you will **not** receive any automated screenings, including emails. This should be either __True__ or __False__. The default is __False__.

> *Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If __False__, you will **not** receive any automated screenings on that day.

Reviewer: ```suggestion
- **ephemeral**: Whether or not to hide the content from everyone else. This is optional and should be either __True__ or __False__. This defaults to __True__.
*Note*: The following arguments represent whether to run the health screening on the specified day. This should be either __True__ or __False__. If False, you will **not** receive any automated screenings on that day.
```

- **sunday**: Defaults to __False__.
- **monday**: Defaults to __True__.
- **tuesday**: Defaults to __True__.
- **wednesday**: Defaults to __True__.
- **thursday**: Defaults to __True__.
- **friday**: Defaults to __True__.
- **saturday**: Defaults to __False__.

> `/generate_once`: Generate a singular health screening without using your information set with `/set_auto`. If set, the device **will** be considered.

Reviewer: ```suggestion

`/generate_once`: Generate a singular health screening without using your information set with `/set_auto`. If set, the device **will** be considered.
```
